i18n/nl/docusaurus-plugin-content-docs/current/staff/coaches/attendance.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 3
+---
+
+# Aanwezigheid
+
+Aanwezigheidsregistratie in Jarvis is een optionele functie die in 2 verschillende modi kan functioneren.
```

Comment:
Aangezien we nog in het begin van de docs zitten, kunnen we een standaard metadata tabel maken bovenaan elke feature pagina met wat basis info als:

Zit achter feature toggle: ja/nee
Feature toggle: .. 
Kan aangezet worden door: locatie administrator / Bit Academy
Modussen (als er meerdere zijn): ...

Reply:
dat tweede - standaard tabel. Dus gewoon in markdown